DiscordBot/Program.cs

```diff
@@ -1,13 +1,19 @@
 ﻿using System.Threading.Tasks;
+using DiscordBot.Commands;
 using DSharpPlus;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.Interactivity;
 
 namespace DiscordBot
 {
     internal class Program
     {
-        private static void Main(string[] args)
+        private static CommandsNextModule commands;
+        private static InteractivityModule interactivity;
```

Comment:
Why are these stored as static variables? It's best to avoid non constant static variables wherever possible, as it makes the code less predictable. I can't see any reason why these need to be stored as variables? 🙂

Reply:
That was the approach in the example, it seems to work without setting them as static variables so I'll do that for now.